impress-org/givewp
# Donor Profile UI is polished

Resolves #5628 

## Description

This PR introduces various minor improvements to the Donor Profile UI. In addition to making colors and margins and various components more consistent with the original Figma designs, the PR introduces some functionality to improve UX (improved loading states, notice when logged in without a Donor account, etc).

## Affects

This PR primarily affects the frontend rendering logic related to Donor Profiles.

## Visuals

<img width="949" alt="Screen Shot 2021-02-25 at 11 33 33 AM" src="https://user-images.githubusercontent.com/5186078/109185121-acd16500-775d-11eb-85a9-949622198ad7.png">

## Pre-review Checklist

## Testing Instructions

For a complete list of which items to test for and check, see the acceptance criteria in the linked issue.
